site/en/blog/show-picker/index.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'Show a browser picker for date, time, color, and files'
+description: >
+  The web platform now ships with a canonical way to show a browser picker.
+authors:
+  - beaufortfrancois
+date: 2022-01-11
+hero: image/vvhSqZboQoZZN9wBvoXq72wzGAf1/zWT5kk3Q5RlDNqGkdgis.jpeg
+alt: A photo of a monthly schedule printed on white paper
+tags:
+  - chrome-99
+---
+
+For a long time, you had to resort to custom widget libraries or hacks to show a
+date picker. The web platform now ships with `HTMLInputElement.showPicker()`, a
```

Comment:
`HTMLInputElement.showPicker` is undefined, because there is no static function with this name. You can use either `HTMLInputElement.prototype.showPicker` or something like `inputElement.showPicker()`.

Reply:
Yeah, anything except something which encourages people to call non-existent methods is a good idea. This is a known issue with MDN in general which I believe they're working on, although at a low priority.